### PR TITLE
DOCS-6916: CSM on Windows

### PR DESCRIPTION
**Pull request by drichards-87**
### What does this PR do? What is the motivation?

DOCS-6916

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

File: content/en/security/cloud_security_management/setup/windows.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
+---
+title: Setting Up Cloud Security Management On Windows
```
Comment (jhgilbert) [inline content/en/security/cloud_security_management/setup/windows.md:2]:
```suggestion
title: Setting up Cloud Security Management on Windows
```
I feel like prepositions aren't usually capitalized in docs titles, but couldn't find anything in the style guide about it.

File: content/en/security/cloud_security_management/setup/windows.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+private: true
+---
+
+<div class="alert alert-warning">Cloud Security Management on Windows is in private beta and should only be installed on hosts that are not critical to production workloads.</div>
+
+Datadog [Cloud Security Management (CSM)][1] on Windows includes built-in threat detection for Windows process and network events. The out-of-the-box Windows ruleset includes the following default rules:
+
+- Certutil used to transmit or decode a file
+- Process memory was dumped using the minidump functions of comsvcs.dll
+- NTDS file referenced in commandline
```
Comment (jhgilbert) [inline content/en/security/cloud_security_management/setup/windows.md:14]:
```suggestion
- NTDS file referenced in command line
```

File: content/en/security/cloud_security_management/setup/windows.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+- Suspicious ntdsutil usage
+- Procdump used to dump process memory
+- Scheduled task created
+- Bitsadmin used to download or execute a file
+- WMI used to remotely execute content
+- Known pentesting tool crackmapexec executed
```
Comment (jhgilbert) [inline content/en/security/cloud_security_management/setup/windows.md:20]:
```suggestion
- Known pen testing tool crackmapexec executed
```

File: content/en/security/cloud_security_management/setup/windows.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+
+1. Download the [Datadog Agent installer][4].
+2. Follow the instructions for command line installation using command prompts or PowerShell.
+
+## Configuration
+
+### Enable CSM
+
+1. Ensure you have access to `C:\ProgramData`, which is a hidden folder.
+    - In **File Explorer**, click the **View** tab, and clear the **Hidden items** checkbox. The **ProgramData** folder should now be visible when navigating to the C: drive. The transparent icon indicates it is a hidden folder.
```
Comment (jhgilbert) [inline content/en/security/cloud_security_management/setup/windows.md:54]:
```suggestion
    - In **File Explorer**, click the **View** tab, and clear the **Hidden items** checkbox. The **ProgramData** folder should now be visible when navigating to the `C:` drive. The transparent icon indicates it is a hidden folder.
```

File: content/en/security/cloud_security_management/setup/windows.md [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+When you enable CSM on Windows, the Agent sends a log to Datadog to confirm that the Windows default ruleset has been successfully deployed. To view the log, navigate to the [**Logs**][7] page in Datadog and search for `@agent.rule_id:ruleset_loaded`.
+
+Another method to verify that the Agent is sending events to CSM is to manually trigger a Windows security signal.
+
+1. In Windows, open a command prompt as Administrator and run the command `schtasks`.
+2. In Datadog, navigate to the [CSM Signals Explorer][8] to view the generated Windows signals.
+    - To view signals originating from configured Windows hosts, filter the signals by hostname using the **Hosts** > **Hostnames** facet.
+    - To filter by Windows rules, use the **Workflow** > **Rule Name** facet.
+
+To get alerts whenever a Windows signal is created, create a [Notification Rule][9] that focuses on the "host" tag specifically for configured Windows hosts.
```
Comment (jhgilbert) [inline content/en/security/cloud_security_management/setup/windows.md:78]:
```suggestion
To get alerts whenever a Windows signal is created, create a [Notification Rule][9] that focuses on the `host` tag specifically for configured Windows hosts.
```

**Review by jhgilbert — APPROVED** [review]
Approved with minor suggestions that are more queries than suggestions. :) Thank you!

File: content/en/security/cloud_security_management/setup/windows.md (hunk 2 above)
Comment (drichards-87) [reply content/en/security/cloud_security_management/setup/windows.md:14]: Good catch. This is actually a rule name, so I'm going to leave as-is and alert the PM that we should update the rule name (and then I'll circle back and update this in the doc).

**Pull request merged**